### PR TITLE
Add Italian Translation

### PR DESCRIPTION
**Pull request by beltra**

### CODE REVIEW

**Review by NicoWeio — CHANGES_REQUESTED** [review]
Thanks a lot for your work! 
I think there is a simple typo, but just to be sure, I'll ask: _uesto_ should be _Questo_, right?

File: language_it-IT.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    SEEK_SUCCESS: "Okay.",
+
+    ERR_NO_TOKEN: "Non hai ancora collegato questa skill al tuo account Spotify. Per farlo, apri l'app Alexa e segui le istruzioni.",
+    ERR_UNKNOWN: "Errore, riprova più tardi.",
+
+    // https://developer.spotify.com/documentation/web-api/reference/object-model/#player-error-object
+    ERR_REASON: {
+      NO_PREV_TRACK: "Questo comando richiede una traccia precendente, ma non ce n'è nessuna nella playlist.",
+      NO_NEXT_TRACK: "uesto comando richiede una traccia successiva, ma non ce n'è nessuna disponibile nella playlist.",
```
Comment (NicoWeio) [inline language_it-IT.js]: Looks like a typo to me
```suggestion
      NO_NEXT_TRACK: "Questo comando richiede una traccia successiva, ma non ce n'è nessuna disponibile nella playlist.",
```

Comment (beltra) [reply language_it-IT.js]: Yeah, definitely a typo

File: language_it-IT.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      NOT_PLAYING_CONTEXT: "Questo comando richiede che una playlist sia attualmente in riproduzione.",
+      ENDLESS_CONTEXT: "La riproduzione casuale non può essere attivata su una playlist senza fine.",
+      CONTEXT_DISALLOW: "Non è stato possibile eseguire questo comando.",
+      // ALREADY_PLAYING: "La traccia non dovrebbe essere ricominciata da capo se la stessa traccia e la stessa playlist sono già in riproduzione e se c'è un punto di ripresa.",
+      // RATE_LIMITED: "Sei al momento limitato per un numero eccessivo di richieste.",
+      REMOTE_CONTROL_DISALLOW: "La riproduzione non può essere controllata.",
+      DEVICE_NOT_CONTROLLABLE: "Non è possibile controllare questo dispositivo da remoto.",
+      VOLUME_CONTROL_DISALLOW: "Non è possibile controllare il volume di questo dispositivo.",
+      NO_ACTIVE_DEVICE: "Non c'è nessun dispositivo disponibile su cui eseguire questo comando.",
+      PREMIUM_REQUIRED: "Sfortunatamente questa richiesta è disponibile solo per gli utenti premium.",
```
Comment (NicoWeio) [inline language_it-IT.js]: Please somehow mention that it's about _Spotify premium_, not e.g. a premium version of this skill. I missed that while translating from German to English.
Let me know if the suggested change below is okay or just commit it yourself. :)
```suggestion
      PREMIUM_REQUIRED: "Sfortunatamente questa richiesta è disponibile solo per gli utenti Spotify premium.",
```

Comment (NicoWeio) [reply language_it-IT.js]: Unresolving since this isn't fixed yet.

Comment (beltra) [reply language_it-IT.js]: Oh sorry, I thought you did fix it

**Review by NicoWeio — APPROVED** [review]
All good now, merging…
Thanks again, and congratulations on your first (merged) PR on GitHub! 🎉

**Pull request merged**